<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>previewwindow</class>
 <widget class="QMainWindow" name="previewwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preview Window</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="animateButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>470</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Animate</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="actualSizeRadio">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>470</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Actual Size</string>
    </property>
   </widget>
   <widget class="QSlider" name="fpsSlider">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>470</y>
      <width>160</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>60</number>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TickPosition::TicksAbove</enum>
    </property>
    <property name="tickInterval">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="fpsLabel">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>470</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>FPS</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="fpsDisplay">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>470</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="intValue" stdset="0">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="spriteLabel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>541</width>
      <height>301</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>fpsSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>fpsDisplay</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>582</x>
     <y>510</y>
    </hint>
    <hint type="destinationlabel">
     <x>504</x>
     <y>509</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>animateButton</sender>
   <signal>clicked()</signal>
   <receiver>previewwindow</receiver>
   <slot>animation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>499</y>
    </hint>
    <hint type="destinationlabel">
     <x>137</x>
     <y>528</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>animation()</slot>
 </slots>
</ui>
